### PR TITLE
merge develop to master - prod deployment 09.05.2020

### PR DESCRIPTION
**Pull request by emina-str**

### CODE REVIEW

File: addressservice.yml [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
-    password: ${infrastructure.database.oracle-egov.password}
-  jpa:
-    database-platform: org.hibernate.dialect.Oracle10gDialect
-    generate-ddl: false
-    show-sql: true
-    hibernate:
-      ddl-auto: validate
+    datasource:
+        initialize: false
+        type: com.zaxxer.hikari.HikariDataSource
```
Comment (birom) [inline addressservice.yml:4]: @seco-mgabor @fankhauserf 
Did somebody else tested the addresses 2 on Staging or on Openshift? Both of them should use the new Database now.

Comment (seco-mgabor) [reply addressservice.yml:4]: I didn't make any tests.

Comment (birom) [reply addressservice.yml:4]: @seco-mgabor 
I did some smoke tests here: http://test-0.apps.admin.arbeitslosenkasse.ch/#/. 
This is the addresses 2 on Openshift. But I do not know if that is enough.
Can you please clarify if we are allowed to change the addresses 2 on prod. 

Thx

Comment (seco-mgabor) [reply addressservice.yml:4]: https://staging.job-room.ch/addresses/#/ is working now. :)

**Pull request merged**